junitparser/cli.py

```diff
@@ -65,39 +70,25 @@ def _parser(prog_name=None):  # pragma: no cover
     )
 
     # command: verify
-    merge_parser = command_parser.add_parser(
+    verify_parser = command_parser.add_parser(  # noqa: F841
```

Comment:
I presume the variable is not needed if unused.
```suggestion
    command_parser.add_parser(
```

Reply:
The `verify_parser` is still relevant and used and might be modified in the future. 

Its variable binding is only unused because it currently only has the attributes inherited from the `abstract_parser` and no explicit ones of its own.

I believe it should be kept for consistency with the other parsers and to increase the readability for future modifications.